Dockerfile

```diff
@@ -56,30 +56,31 @@ FROM gcr.io/distroless/base-debian12:latest AS base
 USER 1000
 
 FROM base AS enduro
-COPY --from=build-enduro --link /out/enduro /home/enduro/bin/enduro
-COPY --from=build-enduro --link /src/enduro.toml /home/enduro/.config/enduro.toml
+COPY --link --from=build-enduro /out/enduro /home/enduro/bin/enduro
+COPY --link --from=build-enduro /src/enduro.toml /home/enduro/.config/enduro.toml
+COPY --link hack/xsd/premis.xsd /home/enduro/premis.xsd
```

Comment:
This line is the main diff, all others are just adding missing `--link` to the `COPY` layers and reorganizing them a little.